HomeUI/src/views/apps/marketplace/AppView.vue

```diff
@@ -735,8 +831,12 @@ export default {
           }
         });
       });
-
       currentComponent.value = props.appData.compose[0];
+      autoSelectNodes().then((v) => {
```

Comment:
this way it would be trigerred on all marketplace apps?
I think there should be first the check enterpriseApp = true before triggering this